packages/react-native/ReactCommon/react/nativemodule/defaults/React-defaultsnativemodule.podspec

```diff
@@ -31,21 +37,31 @@ Pod::Spec.new do |s|
   s.author                 = "Meta Platforms, Inc. and its affiliates"
   s.platforms              = min_supported_versions
   s.source                 = source
+  s.compiler_flags         = folly_compiler_flags
   s.source_files           = "*.{cpp,h}"
   s.header_dir             = "react/nativemodule/defaults"
   s.pod_target_xcconfig    = { "CLANG_CXX_LANGUAGE_STANDARD" => rct_cxx_language_standard(),
                                "HEADER_SEARCH_PATHS" => header_search_paths.join(' '),
+                               "OTHER_CFLAGS" => "$(inherited) " + folly_compiler_flags,
                                "DEFINES_MODULE" => "YES" }
 
   if ENV['USE_FRAMEWORKS']
     s.module_name            = "React_defaultsnativemodule"
     s.header_mappings_dir  = "../.."
   end
 
-  install_modules_dependencies(s)
```

Comment:
Wonder if there should be an internal version of this function that has the rest of these definitions without the ReactCodegen, and this would be used for stuff like third party libraries